example/BUILD.bazel

```diff
@@ -45,3 +47,15 @@ java_binary(
     main_class = "net.sourceforge.pmd.PMD",
     runtime_deps = ["@net_sourceforge_pmd"],
 )
+
+# We can test that it works with:
+# bazel run :ruff -- help
+native_binary(
+    name = "ruff",
+    src = select({
+        "@platforms//os:osx": "@ruff_bin_darwin_arm64//:ruff",
```

Comment:
We'll need more platforms like darwin x86 - we should mirror all the ones the ruff project publishes. Okay for a follow-up though. We should have a script that does the mirroring (and can automate it sending us PRs)